vllm/worker/hpu_model_runner.py

```diff
@@ -412,7 +412,7 @@ def __init__(self, model, vllm_config, is_causal, sampler):
         # This is to ensure that we keeps
         # the static and dynamic parts distinct.
         if htorch.utils.internal.is_lazy():
-            if self.model_is_mrope and hasattr(self.model, 'visual'):
+            if 0:#self.model_is_mrope and hasattr(self.model, 'visual'):
```

Comment:
pls don't hard code the check. This will break other models I believe.

Reply:
Rather than use the model type(Because that needs to import the python file which defines it), I choose to add a flag in that model to indicate that.